ci/add-test-certs.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+sudo mkdir -p /opt/easyengine/nginx/certs
```

Comment:
Remove CI directory since it's not needed for now as you haven't added any tests